src/datatrove/executor/slurm.py

```diff
@@ -111,6 +111,7 @@ def __init__(
         mail_type: str = "ALL",
         mail_user: str = None,
         requeue: bool = True,
+        srun_args: dict = {},
```

Comment:
Convention is usually to pass a sentinel value (None) as default instead of a mutable object